examples/v2/project_creation/apis.py

```diff
@@ -19,14 +19,19 @@ def GenerateConfig(context):
 
   project_id = context.properties['project']
   billing = context.properties['billing']
+  concurrent_api_activation = context.properties['concurrent_api_activation']
 
   resources = []
-  for api in context.properties['apis']:
+  for index, api in enumerate(context.properties['apis']):
+    dependsOn = [project_id, billing]
+    # Serialize the activation of all the apis by making api_n depend on api_n-1
```

Comment:
I thought about did not know where to draw the line. I could perhaps change it  so that the concurrency level is controlled by a parameter and default it to 1 or 2. That would be a pretty involved change and will require unit tests, and prefer to do that in a separate CL.